docs/chain_documentation/approach.md

```diff
@@ -1,18 +1,27 @@
 # Approach
 
-The Geoclimate library is a collection of spatial processes (algorithms) that allows to compute indicators as form factor, densities, sky view factor... based on vector GIS layers. 
-Each indicator is computed and collected for a specific spatial unit. 
-Indicators are then used in statistical classifications to categorize and extract shared properties or organizations for a studied area. 
+The Geoclimate library is a collection of spatial processes (algorithms) that allows to compute indicators such as form factor, densities, sky view factor... based on vector GIS layers. 
 
-The vector GIS layers used by the algorithms must feed an input data model. 
+Computed for a specific spatial unit, each indicators are then used in statistical classifications to categorize and extract shared properties or organizations for a studied area.
 
-This model is organized around a set of thematic layers: building, road, rail, hydrography, vegetation. 
-
-Each layer follows a dictionary of values that define the name and the type of columns and their constraints 
-(min, max value, allowed string value to describe a feature ...). 
+To do so, the vector GIS layers used by the algorithms must feed an input data model. This model is organized around a set of thematic layers (building, road, railway, hydrography, vegetation)  which must follow a dictionary of values defining the name and type of columns as well as their constraints (min / max value, allowed string value to describe a feature, ...). 
```

Comment:
DOn't like that wording.

Reply:
I did not remove any initial ideas. I just merged 3 very strict sentences into 2 and made it more clear (smooth)